### ISSUE TITLE
Restore UI suggests incorrect location

### ISSUE THREAD

**Issue by MarcDiethelm** [schildbach/bitcoin-wallet#233]
The restore dialog presents the following text:
> Pick a wallet backup file from external storage (/storage/emulated/0/Download)

However reading from the suggested path does not work, even if it is created for that purpose manually.
Docs state that the backup files should be located in `/Download`, which does work. Fixing that text would be a very good idea! :)

**Comment by schildbach** [comment]
How do you try to read from that path? What happens if you try to? What "docs" do you mean?

**Comment by MarcDiethelm** [comment]
Yeah, I realized you would probably need some clarification, after looking into this slightly longer. :)

The scenario is you're on a new phone (factory reset or brand new) and need to import your previous wallet backup. I had that backup encrypted and stored in Google Drive... So you enter the restore UI, but no backup is found (though all data backed up with my Google account is already restored). Thus you need to place the wallet backup manually, by copying it from Drive using Android File Transfer (I'm on a Mac yo).

So there you have it: Creating the suggested path `/storage/emulated/0/Download` in Android File Transfer apparently is wrong, probably because the path `/storage/emulated/0` from the user perspective is a hidden prefix?

Trying to read from the path I created, no backup is found, since I now probably just added the prefix path to the existing prefix.

By docs I mean the repository README.

From my perspective the path that I can use is simply `/Download` and this is where I need to place the backup to restore from. If the text had pointed to this path it would have saved me some FUD (again)...

Hope this helps! :)

**Comment by schildbach** [comment]
The path that is printed is simply the result of `Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_DOWNLOADS)`. That's where it looks for backup files. I agree we could mention that if you mount your SD card to your computer, the path to the SD card is irrelevant.

If you have your backup on Google drive you do not need to use Safety > Restore to begin with. Simply tap on the file in Google drive and it will restore from there. That should probably be mentioned in the Safety > Restore dialog but unfortunately it's already too stuffed.

The README mentions `/sdcard/Download/` which points to the download directory on *most* devices (`/sdcard` is a symlink).

**Comment by MarcDiethelm** [comment]
Ok, thanks, now I understand better what's going on. I think it would actually be an improvement to just say that the backup files are stored in `Downloads` on the *SD card*. The app already looks in the correct place and if a user actually is moving stuff around on the phone's file system I assume that he knows how to find the external storage. And yes that would also close this issue.

Now the info that the backup file in Drive is associated with Bitcoin Wallet and tappable (in recent versions, older backups apparently don't do this) is extremely helpful and would have saved me the hassle in the first place... Maybe, if you feel there is no space to mention this in the dialog you could provide a link instead, with some more verbose instructions?

**Comment by schildbach** [comment]
Ok. With the next version, the path to external storage will be cut from the backup paths shown in the UI. As most people will access their external storage via mounted filesystem, that's a reasonable thing to do. And those people using adb to log in to their device, will likely know that their external storage is mounted under /sdcard locally.

Note I still do not want to use the word SD card, as most devices do not have an SD card. If only there was a way to determine the type of card used (or fixed memory)…

The thing about being to able to open backup files by tapping on them is it should be obvious. That metaphor is established since 1973, and the only thing that has changed since is people use their finger instead of the mouse.

**Comment by MrMatinV** [comment]
Hello World i think we must change something in the world, world health need backup
